buildenv/jenkins/docker-slaves/x86/ubuntu16/Dockerfile

```diff
@@ -103,5 +103,40 @@ RUN mkdir /var/run/sshd \
 # SSH login fix. Otherwise user is kicked off after login
 RUN sed 's@session\s*required\s*pam_loginuid.so@session optional pam_loginuid.so@g' -i /etc/pam.d/sshd
 
+# Install OpenSSL v1.1.1b
+# Required for JITaaS
+RUN cd /tmp \
+  && wget https://github.com/openssl/openssl/archive/OpenSSL_1_1_1b.tar.gz \
+  && tar -xzf OpenSSL_1_1_1b.tar.gz \
+  && rm -f OpenSSL_1_1_1b.tar.gz \
+  && cd /tmp/openssl-OpenSSL_1_1_1b \
+  && ./config --prefix=/usr/local/openssl-1.1.1b --openssldir=/usr/local/openssl-1.1.1b \
+  && make \
+  && make install \
+  && cd .. \
+  && rm -rf openssl-OpenSSL_1_1_1b \
+  && echo "/usr/local/openssl-1.1.1b/lib" > /etc/ld.so.conf.d/openssl-1.1.1b.conf \
+  && ldconfig
+#  && echo "export PATH=/usr/local/openssl-1.1.1b/bin:$PATH" > /etc/profile.d/openssl-1.1.1b.sh
+ENV PATH="/usr/local/openssl-1.1.1b/bin:${PATH}"
```

Comment:
I'm not entirely sure, but I think this only sets the path variable for the root user, so the jenkins user won't have the updated path and would continue to use the old openssl.